lib/hammer/application.ex

```diff
@@ -7,19 +7,27 @@ defmodule Hammer.Application do
   require Logger
 
   def start(_type, _args) do
-    {backend_module, backend_config} = Application.get_env(
+    start_backend(:sup, Application.get_env(
       :hammer,
       :backend,
       {Hammer.Backend.ETS, []}
-    )
+    ))
+
+  end
+
+  defp start_backend(_key, which) when is_list(which) do
```

Comment:
Do we also need an `is_tuple` function-head to handle the default case? (when the config is just a single backend tuple)?

Reply:
Also, could you add a comment around here, showing the expected formats for backend configurations?

Reply:
About the is_tuple: I think that depends if an error because of a missing tuple will be clear enough for the end-user to figure out what's wrong.